tdrs-backend/tdpservice/parsers/fields.py

```diff
@@ -1,10 +1,19 @@
 """Datafile field representations."""
 
 import logging
-from .validators import value_is_empty
 
 logger = logging.getLogger(__name__)
 
+def value_is_empty(value, length):
+    """Handle 'empty' values as field inputs."""
+    empty_values = [
+        '',
+        ' '*length,  # '     '
+        '#'*length,  # '#####'
+        '_'*length,  # '_____'
+    ]
+
+    return value is None or value in empty_values
 
```

Comment:
I had to add value_is_empty to a couple of files where I needed to use to resolve circular import. I am fixing this issue